syft/frameworks/torch/__init__.py

```diff
@@ -2,6 +2,5 @@
 from . import federated
 from . import differential_privacy
 from . import crypto
-from . import pointers
```

Comment:
While we're here could we make these absolute as well 😉

Reply:
as above 👨‍🎨